### DISCUSSION TITLE
Do the built-in jinja-filters work with this plugin?

### DISCUSSION THREAD

**Discussion by dpapagiannis** [fralau/mkdocs-macros-plugin#253] [Q&A] [answered]
Hello and thank you for this extremely useful plugin.

I have a question regarding the [built-in filters that Jinja2 has](https://jinja.palletsprojects.com/en/stable/templates/#list-of-builtin-filters). I tried using them (e.g., `{{35/2|round|int}}`) but they don't seem to work out of the box.

Am I missing something?

PS: In my config I'm just using:

```yaml
plugins:
  - search
  - macros
```

PS2: I didn't find a similar discussion here, which surprised me.

**Comment by fralau** [comment, marked as answer]
Put parentheses around 35/2 ?

This worked for me: `{{ (-35.0/2) | abs }}`

**Reply by dpapagiannis** [reply]
I'm bamboozled, now it works. The thing is I had already put parentheses, but it was not working, at least yesterday(!).

Anyway, sorry for the noise, it was probably an `interface chaise–clavier` error. :sweat_smile:

**Comment by fralau** [comment]
Well that was an opportunity to test it for myself 👍 
I added a unit test for that.
